lib/jekyll-admin/server/page.rb

```diff
@@ -34,6 +35,24 @@ class Server < Sinatra::Base
 
       private
 
+      def ensure_html_content
+        unless html_content?
```

Comment:
I believe this can be better implemented as a guard statement, e.g., `return if html_content?`